Catappult/appcoins-wallet-android
# [APPC-2240] - Added E-Voucher details screen.

**What does this PR do?**

Added new screen E-Voucher Details.

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] EVoucherDetailsFragment.kt

**How should this be manually tested?**

Go to promotions screen, you should see a button in the top left corner that will take you to E-Voucher Details screen.

**What are the relevant tickets?**

  Tickets related to this pull-request: [MOB-APPC-2240](https://aptoide.atlassian.net/browse/APPC-2240)

**Questions:**

New strings were added that need translations.




**Code Review Checklist**
